### ISSUE TITLE
Deploy releases

### ISSUE THREAD

**Issue by digitalhuman** [ayufan-rock64/linux-build#294]
Hi there,

Maybe a stupid question but how does one deploy one of the releases? Were can I find documentation for this?

Cheers,

Victor

**Comment by TrashCanMan** [comment]
I never tried it myself but i think that you need to run [rock64_install_to_emmc.sh](https://github.com/ayufan-rock64/linux-build/pull/71/commits/2d06fac60fa2f038585d01a04ea4f65282d9f0ce) in the CLI

**Comment by digitalhuman** [comment]
Yes but those don't support the Debian rock64 release. That is my point.

**Comment by digitalhuman** [comment]
So I now have the USB adapter. Still no luck flashing eMMC. Windows shows its only 14GB while its 64GB. Linux (CentOS) doesn't recognize the storage/usb adapter. Any idea?

**Comment by wilmardo** [comment]
@digitalhuman Have you tried [Etcher](https://www.balena.io/etcher/)?

**Comment by digitalhuman** [comment]
@wilmardo yeah I did. No luck, not even with one of the compatible images. It does write, but after validation it returns "Validation failed".
